x-pack/plugins/apm/public/components/app/ServiceMap/Cytoscape.tsx

```diff
@@ -80,7 +80,7 @@ function getLayoutOptions(
     roots: selectedRoots.length ? selectedRoots : undefined,
     fit: true,
     padding: nodeHeight,
-    spacingFactor: 0.85,
+    spacingFactor: 1.2,
```

Comment:
Seems like a magic number but if it works...

Reply:
The docs for this setting (https://js.cytoscape.org/#layouts/breadthfirst) describe it as:
> positive spacing factor, larger => more space between nodes

Reply:
All right 👍